src/cpu/gemm/gemm.cpp

```diff
@@ -222,6 +247,23 @@ dnnl_status_t gemm_s8x8s32(const char *transa, const char *transb,
                 LDA, ao, B, LDB, bo, beta, C, LDC, co);
 #endif
 
+#if DNNL_PPC64
+#ifdef __MMA__
+    int ATflag = (*transa == 'T') || (*transa == 't');
+    int BTflag = (*transb == 'T') || (*transb == 't');
+    int m = (int)*M;
+    int n = (int)*N;
+    int k = (int)*K;
+    int lda = (int)*LDA;
+    int ldb = (int)*LDB;
+    int ldc = (int)*LDC;
+
+    return cblas_gemm_s8u8s32_ppc64(ATflag, BTflag, offsetc, m, n, k, *alpha, A,
```

Comment:
Are you sure this would work as expected?
In this function, B is of type `const int8_t *` and you cast it to `const uint8_t*`.
I would expect that you will get wrong results for some values. See this example with [godbold](https://godbolt.org/z/x3vocfoxh).

Reply:
My apologies.
I should have been more generous with my comments.

I chose to have one routine that handles both signed and unsigned B.
I manage that through the use of a flag at the end of the call list.

The called routine knows that if the flag is 1, the B matrix actually contains signed values, 
not unsigned, and it processes it accordingly.

At the very least, I will add comments to clarify this.

OR, I *could* replace this routine with two separate routines to handle signed and unsigned versions of B explicitly, but the flag is fairly easy to understand when you know to look for it.

Inside src/cu/ppc64/ppc64_gemm_s8x8s32.cpp, look for "flipB_flag".

If you're wanting me to create two routines rather than one with a flag, let me know and I'll do that, but my default plan now is to add comments here to explain the behavior.